src/components/language/language.service.ts

```diff
@@ -388,65 +381,119 @@ export class LanguageService {
       session.userId = this.config.anonUser.id;
     }
 
-    const props = [
-      'name',
-      'displayName',
-      'isDialect',
-      'populationOverride',
-      'registryOfDialectsCode',
-      'leastOfThese',
-      'leastOfTheseReason',
-      'displayNamePronunciation',
-      'sensitivity',
-      'sponsorDate',
-    ];
-
-    const baseNodeMetaProps = ['id', 'createdAt'];
-
-    const childBaseNodeMetaProps: ChildBaseNodeMetaProperty[] = [
-      {
-        parentBaseNodePropertyKey: 'ethnologue',
-        parentRelationDirection: 'out',
-        childBaseNodeLabel: 'EthnologueLanguage',
-        childBaseNodeMetaPropertyKey: 'id',
-        returnIdentifier: 'ethnologueLanguageId',
-      },
-    ];
+    /*
+          MATCH (requestingUser:User { active: true, id: 'rootadminid' })
+          MATCH (node:Language { active: true })
+          WHERE node.id = 'YszZRgDw-'
+          MATCH (requestingUser)<-[:member*1..]-(:SecurityGroup { active: true })-[:permission]->(perms:Permission { active: true })-[:baseNode]->(node)
+          with collect(perms) as permList, node
+          MATCH (node)-[{active: true}]->(props:Property {active: true})
+          with collect(props) as propList, permList, node
+          return permList, propList, labels(node)
+     */
 
     const query = this.db
       .query()
       .call(matchRequestingUser, session)
-      .call(matchUserPermissions, 'Language', langId)
-      .call(addAllSecureProperties, ...props)
-      .call(addAllMetaPropertiesOfChildBaseNodes, ...childBaseNodeMetaProps)
-      .with([
-        ...props.map(addPropertyCoalesceWithClause),
-        ...childBaseNodeMetaProps.map(addShapeForChildBaseNodeMetaProperty),
-        ...baseNodeMetaProps.map(addShapeForBaseNodeMetaProperty),
-        'node',
+      .match([node('node', 'Language', { active: true, id: langId })])
```

Comment:
Nitpick: give a more specific name for 'node', maybe 'languageNod'.  

I'm just reading this PR to understand the new query changes but betting labeling would help a beginner like me follow along

Reply:
I believe `node` almost always represents the thing you are fetching. We are in the `LanguageService` so it represents a language.

Reply:
its convention used in the `query.helper.ts` file. It must be `node`.

Reply:
got it, thanks guys